client/my-sites/plan-features/header.jsx

```diff
@@ -3,23 +3,33 @@
  */
 import React, { Component, PropTypes } from 'react';
 import noop from 'lodash/noop';
+import identity from 'lodash/identity';
 
 /**
  * Internal Dependencies
  **/
 import { localize } from 'i18n-calypso';
 import Gridicon from 'components/gridicon';
+import PlanFeaturesPrice from './price';
 import {
 	PLAN_FREE,
 	PLAN_PREMIUM,
 	PLAN_BUSINESS
 } from 'lib/plans/constants';
 
-class PlanFeaturesHeader extends Component {
+export class PlanFeaturesHeader extends Component {
```

Comment:
Why do we need to `export` `PlanFeaturesHeader` without running it through `localize`? What's the use case?